packages/components/src/box-control/utils.ts

```diff
@@ -7,7 +7,42 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { parseQuantityAndUnitFromRawValue } from '../unit-control/utils';
-import type { BoxControlProps, BoxControlValue } from './types';
+import type {
+	BoxControlProps,
+	BoxControlValue,
+	CustomValueUnits,
+} from './types';
+
+export const CUSTOM_VALUE_SETTINGS: CustomValueUnits = {
+	px: { max: 300, steps: 1 },
```

Comment:
Nit: Shouldn't these be `step` (singular)?

```suggestion
	px: { max: 300, step: 1 },
```

Reply:
I think that makes more sense. I initially grabbed this from the component we've been emulating a bit for this change: https://github.com/WordPress/gutenberg/blob/84715c01148344c318344978508b5d442526ed0c/packages/block-editor/src/components/spacing-sizes-control/input-controls/spacing-input-control.js#L34-L63